### ISSUE TITLE
Using an expression to set a @on{EVENT}:preventDefault value in an SVG throws exception when expression changes from true to false

### ISSUE THREAD

**Issue by NickBarrett** [dotnet/aspnetcore#25297]
I realise for now, SVG support is limited in Blazor (#18271) but I wanted to add this here so that it can be reviewed when the team has time.

### Describe the bug
Adding `@on{EVENT}:preventDefault` (or `@on{EVENT}:stopPropagation`) to an `<svg>` tag, and setting the value of the attribute using an expression, causes a "Microsoft.JSInterop.JSException: Cannot remove attribute from non-element child
Error: Cannot remove attribute from non-element child" exception to be thrown when the value of the expression changes from `true` to `false`.

### To Reproduce
```
@page "/"

<svg width="100%" height="100%" style="background-color: green" @onpointerdown:preventDefault="_preventDefaultOnPointerDown" @onpointerdown="OnPointerDown"></svg>

@code {
    private bool _preventDefaultOnPointerDown = false;

    private Task OnPointerDown(PointerEventArgs e)
    {
        _preventDefaultOnPointerDown = !_preventDefaultOnPointerDown;
        return Task.CompletedTask;
    }
}
```

The above code switches the value of `_preventDefaultOnPointerDown` when when you click on the green svg. As the initial state is `false`, the first switch works fine. When you click the svg for the second time though, `_preventDefaultOnPointerDown` returns to `false` and the exception is thrown.

Using `console.dir($0)` in the browser console (Chrome) I can see that the initial state of the `<svg>` tag has the following attributes:
_blazorEvents_1: e
  preventDefaultFlags: null

When `_preventDefaultOnPointerDown` is set to `true`, this changes to 
_blazorEvents_1: e
  preventDefaultFlags:
    pointerdown: true

I don't know what is happening to `preventDefaultFlags` when we set `_preventDefaultOnPointerDown` back to `false`, but I would assume it's possibly remving it rather then setting `pointerdown: false`.

### Exceptions (if any)
Microsoft.JSInterop.JSException: Cannot remove attribute from non-element child
Error: Cannot remove attribute from non-element child
    at e.applyEdits (http://localhost:5000/_framework/blazor.webassembly.js:1:14050)
    at e.updateComponent (http://localhost:5000/_framework/blazor.webassembly.js:1:12880)
    at Object.t.renderBatch (http://localhost:5000/_framework/blazor.webassembly.js:1:1704)
    at Object.window.Blazor._internal.renderBatch (http://localhost:5000/_framework/blazor.webassembly.js:1:34784)
    at _mono_wasm_invoke_js_unmarshalled (http://localhost:5000/_framework/wasm/dotnet.3.2.0.js:1:172099)
    at wasm_invoke_iiiiii (http://localhost:5000/_framework/wasm/dotnet.wasm:wasm-function[3160]:0x9b33d)
    at icall_trampoline_dispatch (http://localhost:5000/_framework/wasm/dotnet.wasm:wasm-function[5777]:0xfe711)
    at mono_wasm_interp_to_native_trampoline (http://localhost:5000/_framework/wasm/dotnet.wasm:wasm-function[4607]:0xca81d)
    at ves_pinvoke_method (http://localhost:5000/_framework/wasm/dotnet.wasm:wasm-function[3209]:0x9cd40)
    at interp_exec_method (http://localhost:5000/_framework/wasm/dotnet.wasm:wasm-function[1120]:0x2598d)
  at Microsoft.JSInterop.WebAssembly.WebAssemblyJSRuntime.InvokeUnmarshalled[T0,T1,T2,TResult] (System.String identifier, T0 arg0, T1 arg1, T2 arg2) <0x2cdad80 + 0x00046> in <filename unknown>:0 
  at Microsoft.JSInterop.WebAssembly.WebAssemblyJSRuntime.InvokeUnmarshalled[T0,T1,TResult] (System.String identifier, T0 arg0, T1 arg1) <0x2cdaca0 + 0x00014> in <filename unknown>:0 
  at Microsoft.AspNetCore.Components.WebAssembly.Rendering.WebAssemblyRenderer.UpdateDisplayAsync (Microsoft.AspNetCore.Components.RenderTree.RenderBatch& batch) <0x2cdaba8 + 0x0001e> in <filename unknown>:0 
  at Microsoft.AspNetCore.Components.RenderTree.Renderer.ProcessRenderQueue () <0x2b8e9d0 + 0x000f2> in <filename unknown>:0 

### Further technical details
This problem is present in both current ASP.Net Core (3.1)
- ASP.NET Core runtime version: Microsoft.AspNetCore.App 3.1.7
- IDE: VS 16.7.2

and latest preview of ASP.Net 5
- ASP.NET Core runtime version: Microsoft.AspNetCore.App 5.0.0-preview.8.20414.8
- IDE: VS 16.8.0 Preview 2.0

### Some thoughts
While SVG support is limited in Blazor, we are still using it rather extensively. We are writing our own custom visual components and our current test app has roughly 12k <svg>, <rect>, <g>, <line>, and <text> tags in the DOM at once. So far, the initial render can take a bit (a couple of seconds) but after that, we've been able to optimise quite a bit to get good performance out of our test app. We're looking forward to what the team brings in the future 👍

**Comment by TanayParikh** [comment]
Putting this back into backlog due to limited community interest at this time.

**Comment by t-schreibs** [comment]
I'm running into what appears to be the same issue. I've got a collection of child `<svg>` elements with `draggable=@CanBeDragged`, and when I change CanBeDragged from, say, true to false in one of them, the same "Cannot remove attribute from non-element child" error is thrown.

**Comment by Sibusten** [comment]
Is this issue also fixed by #43957?

**Comment by NickBarrett** [comment]
I've just tested my repro code and yes, the code now works without the exception being thrown 👍
Thanks @Sibusten for letting me know, and thanks to those involved with PR #43957 to fix the issue.
